app/services/websites/create.rb

```diff
@@ -0,0 +1,56 @@
+module Websites
+  class Create
+    # From the controller, use it like this:
+    #
+    # Websites::Create.call(params, user)
+    def self.call(params, user)
+      new(params, user).call
+    end
+
+    def initialize(params, user)
+      @params = params
+      @user = user
+      @website = Website.where(url: params[:website][:url])
+    end
+
+    def call
+      @descript = create_website
+      update_statistics
+      find_or_create_user
+      fetch_rank
+      @descript
+    end
+
+    
+    private
+
+    attr_reader :params, :user, :website
+
+    def create_website
+      @website = website.first_or_create
+       CollectionWebsite.create(collection_id: collection_id, website_id: website.id)
+
+      @descript = Website.fetch_metadescription(domain: params[:website][:url],website: website)
+      @descript
+    end
```

Comment:
Why is this instance variable being overriden? Please avoid creating instance vars unless it's really necessary. Also, as mentioned above, the variable name is hard to understand.

If the value returned at the end of `self.call` is being used in the controller, please add documentation comments above the method, mentioning what is being returned.